src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneSwitchConfig.java

```diff
@@ -0,0 +1,28 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.Accordion;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneSwitchConfig {
```

Comment:
please use consistent upper or lower case of S in switch
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/be17ba4c-9da6-415f-9641-c25bc997b0e0)

Reply:
Will do, had this from when it was still "Laneswitch" instead of "Lane Switch"